### DISCUSSION TITLE
Translations?

### DISCUSSION THREAD

**Discussion by GM-Ricker** [JamesTKhan/Mundus#252] [Ideas]
Do you plan to add translations? I could contribute to the translation of my local language, it would be really useful

**Comment by JamesTKhan** [comment]
I appreciate the offer! My main concern is that new features and changes are still happening frequently, and that means any time some one updates or changes any text, translations would also need to be created, else they would quickly fall out of date.

Once the editor reaches a more stable state (and ideally more supporters), I could see having translations added, but right now I think it would not be maintainable. I am open to other feedback though.
